node/utils.go

```diff
@@ -310,13 +311,22 @@ func createVolumeDir(hostPath, containerSpec, imageSpec, userSpec, permissionSpe
 	createVolumeDirMutex.Lock()
```

Comment:
sorry, 1m30s is for this https://github.com/control-center/serviced/pull/1547, not https://github.com/control-center/serviced/pull/1546